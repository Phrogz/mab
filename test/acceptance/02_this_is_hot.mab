p( "Test#2" )
p( 40 +( 2 ) )
Lawn p( "Hello World", "This is hot!" )
