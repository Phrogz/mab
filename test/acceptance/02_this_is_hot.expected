Test#2
42
Hello World
This is hot!